src/atoms/TextArea/TextArea.tsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+
+import {
+  Container,
+  Errors,
+  Label,
+  Subtext,
+  TextAreaInput
+} from './TextArea.style';
+
+interface Props {
+  id?: string;
+  name: string;
+  rows: number;
+  cols: number;
+  placeholder?: string;
+  value?: string;
+  label: string;
+  showLabel?: boolean;
+  required?: boolean;
+  onChange?: Function;
+  autocomplete?: string;
+  errors?: Array<string>;
+  subtext?: string;
+}
+
+export type TextAreaProps = Props & React.InputHTMLAttributes<string>;
```

Comment:
```suggestion
export type TextAreaProps = Props & React.TextareaHTMLAttributes<HTMLTextAreaElement>;
```
I think the generic inside these html attribute types is supposed to be the type of element, not the type of input (string). Sorry for not noticing this earlier, we should fix this in all the other places we did this too.

Also, because we are typing it like this, a lot of the props that are actually textarea attributes are already in `otherProps` and properly typed. We do not need to type things like `cols` and `rows` ourselves because `React.TextareaHTMLAttributes<HTMLTextAreaElement>` should do it for us.